tests/utils/iotrace.py

```diff
@@ -8,244 +8,471 @@
+        :param shortcut: Use shorter command
+        :type trace_path: str
+        :type shortcut: bool
+        :return: trace statistics
+        :raises Exception: if statistics are invalid
+        """
+        command = 'iotrace' + (' -P' if shortcut else ' --trace-parser')
+        command += ' -S' if shortcut else ' --statistics'
+        command += (' -p ' if shortcut else ' --path ') + f'{trace_path}'
+        command += (' -f ' if shortcut else ' --format ') + 'json'
+
+        output = TestRun.executor.run(command)
+        if output.stdout == "":
+            raise CmdException("Invalid traces")
+
+        return parse_json(output.stdout)
+
+    @staticmethod
+    def remove_traces(prefix: str, force: bool = False, shortcut: bool = False):
+        """
+        Removes record of the most recent trace
+
+        :param prefix: Trace path or trace path's prefix when ended with '*'
+        :param force: force removal
+        :param shortcut: Use shorter command
+        :type prefix: str
+        :type force: bool
+        :type shortcut: bool
+        :raises Exception: if removal is invalid
+        """
+        command = 'iotrace' + (' -M' if shortcut else ' --trace-management')
+        command += ' -R' if shortcut else ' --remove-traces'
+        command += (' -p ' if shortcut else ' --prefix ') + f'{prefix}'
+
+        if force:
+            command += ' -f ' if shortcut else ' --force '
+
+        error_output = parse_json(TestRun.executor.run(command).stderr)
+        if error_output[0]['trace'] == "No traces removed.":
+            raise CmdException("Invalid traces removal")
+
+    @staticmethod
+    def set_trace_repository_path(trace_path: str, shortcut: bool = False):
+        """
+        :param trace_path: trace path
+        :param shortcut: Use shorter command
+        :type trace_path: str
+        :type shortcut: bool
+        :raises Exception: if setting path fails
+        """
+        command = 'iotrace' + (' -C' if shortcut else ' --trace-config')
+        command += ' -S ' if shortcut else ' --set-trace-repository-path '
+        command += (' -p ' if shortcut else ' --path ') + f'{trace_path}'
+
+        error_output = parse_json(TestRun.executor.run(command).stderr)[0]["trace"]
+        if error_output == "No access to trace directory":
+            raise CmdException("Invalid setting of the trace repository path")
+
+    @staticmethod
+    def help(shortcut: bool = False) -> str:
+        """
+        :param shortcut: Use shorter command
+        :type shortcut: bool
+        :return: io-tracer help
+        :raises Exception: if command fails
+        """
+        return TestRun.executor.run_expect_success(
+            'iotrace' + (' -H ' if shortcut else ' --help ')).stdout
+
+    @staticmethod
+    def version(shortcut: bool = False) -> dict:
+        """
+        :param shortcut: Use shorter command
+        :type shortcut: bool
+        :return: io-tracer version
+        :rtype: dictionary
```

Comment:
Please also add expected key values to doc - because this is not clear, even looking at implementation